package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "typescript-for-nodejs",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "build": "tsc && cpy src/views/*.hbs dist/views",
```

Comment:
You should use `npx tsc` to run modules, since we are not installing the `tsc` as global

Reply:
@nestorvanz Actually,`npx` is not required in npm scripts.

See [npm-run-script](https://docs.npmjs.com/cli/v6/commands/npm-run-script#description):
> In addition to the shell's pre-existing PATH, npm run adds node_modules/.bin to the PATH provided to scripts. Any binaries provided by locally-installed dependencies can be used without the node_modules/.bin prefix.

Also, see https://github.com/npm/npx#examples and [Introducing npx: an npm package runner](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b)